interchain/framework-clone-testing/src/common.rs

```diff
@@ -0,0 +1,63 @@
+use abstract_client::{AbstractClient, Environment};
+use abstract_interface::{Abstract, VCExecFns};
+use cosmwasm_std::{coins, Addr};
+use cw_orch::prelude::*;
+use cw_orch::{contract::Deploy, daemon::ChainInfo};
+use cw_orch_clone_testing::CloneTesting;
+
+/// Returns a shared tokio runtime for all tests
+pub fn rt() -> &'static tokio::runtime::Runtime {
+    lazy_static::lazy_static! {
+        static ref RT: tokio::runtime::Runtime = tokio::runtime::Builder::new_multi_thread()
+        .enable_all()
+        .build()
+        .expect("Should create a tokio runtime");
+    }
+    &RT
+}
+
+/// Sets up the CloneTesting for chain.
+/// Returns the abstract deployment and sender (=mainnet admin)
+pub fn setup(
+    chain: ChainInfo,
+    sender: &str,
```

Comment:
If this function only works for valid Addr, this should be Addr, not &str to prevent mis-use.

Reply:
You can use lazy static to create the addr at compile time.

Reply:
> You can use lazy static to create the addr at compile time.

We use addresses of abstract admins here. We could determine it from querying one of the abstract core contract and remove this field then, would you prefer that instead?

Reply:
>We could determine it from querying one of the abstract core contract and remove this field then, would you prefer that instead?

But that would require "loading" abstract twice, because we can't edit `Chain` of `Abstract`, but I guess it's fine, because we only load local "state"